backend/test/test_users.py

```diff
@@ -801,15 +800,21 @@ def test_user_emails_endpoint_superuser(admin_auth_headers, default_org_id):
 
     for user in user_emails:
         assert user["email"]
+        assert "id" not in user
+        assert "is_superuser" not in user
+        assert user["is_verified"] == True
         orgs = user.get("orgs")
         if orgs == []:
             continue
 
         for org in orgs:
-            assert org["id"]
+            assert "id" not in org
```

Comment:
```suggestion
            assert org["id"]
```
I think this might be a mistake - our models don't keep the org id out of the response (and I'd expect the org id is helpful).